### PR TITLE
(IO-1) Importing command from text

### PR DESCRIPTION
**Pull request by JakubMatuszakHTD**
Authors:
@JakubMatuszakHTD 
@PiotrJurek240693

### CODE REVIEW

**Review by kowallus — CHANGES_REQUESTED** [review]
Bad PR:
- no test, no examples!
- function not implemented in the app (e.g., command manager window)
- poor design, no SRP (one method to read from file & convert from text to commands List)
- ComplexCommand abstraction not used.

File: .idea/.gitignore
Comment (kowallus) [inline .idea/.gitignore:1]: commits polluted with IDE files. Remove .idea folder!

File: Rectangle.txt
Comment (kowallus) [inline Rectangle.txt:1]: such files should be in src/test/resources folder (or subfolder)

File: src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import java.util.List;
+
+public class LoadCommand {
+
+    public static CompoundCommand loadCommandsFromFile(String fileName) {
+        List<String> lines = readLinesFromFile(fileName);
+        List<DriverCommand> commands = new ArrayList<>();
+
+        for (String line : lines) {
+            commands.add(getCommandFromLine(line));
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java:22]: Format of import fixed. No OCP: adding new format (e.g., json) cannot reuse this code without modification.

File: src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LoadCommand {
+
+    public static CompoundCommand loadCommandsFromFile(String fileName) {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java:17]: Importing tangled with a specified datasource (file). Cannot import for (e.g. URL or text field) reusing this code (without modifying it). No SRP and OCP!

**Review by kowallus — CHANGES_REQUESTED** [review]
Bad design

File: .idea/.gitignore
Comment (JakubMatuszakHTD) [reply .idea/.gitignore:1]: Done

File: Rectangle.txt
Comment (JakubMatuszakHTD) [reply Rectangle.txt:1]: Done

File: src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java (hunk 1 above)
Comment (JakubMatuszakHTD) [reply src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java]: Done

File: src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java (hunk 2 above)
Comment (JakubMatuszakHTD) [reply src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java]: Done

**Comment by kowallus** [comment]
Please make new PR to **dev-io-1** branch. Insure there are no conflicts.

**Pull request closed without merging**